### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elenananana**

### CODE REVIEW

File: README.md
Comment (elenananana) [inline README.md:1]: Delete developer release at the end of the paragraph. it's in there twice.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 # Introduction
 
-**Status**: *Alpha, do not use in production*
+**Status**: **Developer Preview** This initial release is a Developer Preview of the Stacks x Pyth integration. The integration is live on both testnet and mainnet and is meant primarily for developers to test, give feedback and ensure reliability and stability of the integration.Developer Release*
```
Comment (LakshmiLavanyaKasturi) [inline README.md:17]:
```suggestion
**Developer Preview** This initial release is a Developer Preview of the Stacks x Pyth integration. The integration is live on both the testnet and mainnet to help developers test, give feedback, and ensure the reliability and stability of the integration. Developer Release*
```

Comment (LakshmiLavanyaKasturi) [reply README.md:17]: Are we adding a release version here? Wondering why we have "Developer Release" at the end of the sentence.

File: README.md
Comment (LakshmiLavanyaKasturi) [reply README.md:1]: @elenananana, You can use the commit suggestion to add the changes.

**Pull request merged**